problems/multiplesOfThreeOrFive.js

```diff
@@ -24,8 +24,15 @@
 * Notice that 15 is a multiple of 3 and 5 BUT it is only added ONCE.
 */
 
-function multiplesOfThreeOrFive() {
-
+function multiplesOfThreeOrFive(n) {
+    let sum = n > 0;
+        sum = n < 10
+    for (let i = 0; i <= n; i++) {
```

Comment:
In we wanted to get the number up to the given number but not including the number that is given. So instead the condition will not be `i <= n`